dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/BrandingUtil.java

```diff
@@ -23,6 +24,20 @@ public static String getSupportTeamName(InternetAddress systemAddress, String ro
         return BundleUtil.getStringFromBundle("contact.support", Arrays.asList(saneDefault));
     }
 
+    public static String getSupportTeamName(InternetAddress systemAddress, String rootDataverseName, Locale locale) {
```

Comment:
Is method without Locale working correctly (we don't want locale in every instance)?.

Reply:
They both work correctly. The distinction is: email notifications are specific to language specified by user in his account, while notifications in the app itself are based on currently selected language.

Hence we need both methods: the one with locale is for emails, the one without is for app notifications.